server/api/applicantSkills.js

```diff
@@ -2,10 +2,22 @@ const express = require("express");
 const router = express.Router();
 
 const {
-  getSkillsForApplicantProfile
+  getSkillsForApplicantProfile,
+  getSkillsForEditApplicantProfile
 } = require("../services/database/applicantSkills");
 
-router.get("/:id/skills", (req, res) => {
+router.get("/getSkillsId/:id", (req, res) => {
```

Comment:
Again, there is no need for this endpoint already there one to get applicant skills by applicant ID.

Reply:
yea I have the function that gets the skills as a name but I need the id here